Formula/dotnet.rb

```diff
@@ -51,6 +51,23 @@ def install
     (bin/"dotnet").write_env_script libexec/"dotnet", DOTNET_ROOT: libexec
   end
 
+  def caveats
+    <<~EOS
+      To ensure the Homebrew installed version of .NET Core is used, you have to
+      add the following line to your ~/.bash_profile, ~/.zshrc or
+      ~/.config/fish/config.fish file:
+
+        export DOTNET_ROOT="#{opt_libexec}"
+
+      If you haven't already, you also need to add the path to the dotnet
+      executable to your PATH environment variable in the same file:
+
+        export PATH="#{HOMEBREW_PREFIX/"bin"}:$PATH"
+
+      Remember to source the file to update your current session.
```

Comment:
This is part of the Homebrew install script, it doesn't add much as a caveat in specific software.

Reply:
Ok, fair enough. I'll remove it.

Reply:
Shell profiles are also often not idempotent (especially for users who need be reminded about setting their `PATH`...), so I'd really rather not display this message.